app/models/project.rb

```diff
@@ -35,4 +37,13 @@ def add_default_photo
                                  image_file_size: 300)
     self.photos = [default_photo] unless photos.count > 0
   end
+
+  def add_default_repayment_rate
+    self.repayment_rate = 26
+  end
+
+  def add_default_repayment_begin
+    self.requested_by = Date.new(2020, 12, 12)
```

Comment:
Yes, eventually, I have a waffle ticket #150 open to address this issue. I didn't want to add that functionality here because this PR was already changing 9 files.